Formula/m/micromamba.rb

```diff
@@ -48,29 +49,32 @@ def install
     args = %W[
       -DBUILD_LIBMAMBA=ON
       -DBUILD_SHARED=ON
-      -DBUILD_MICROMAMBA=ON
```

Comment:
I think you need to set `-DBUILD_MAMBA=ON`.

This should hopefully fix the macOS failures. Not sure about Linux.